_DeploymentAndDistroPackaging/ansible/roles/ciao-controller/defaults/main.yml

```diff
@@ -56,24 +56,24 @@ ciao_openstack_services:
   - service: cinderv2
     type: volumev2
     description: CIAO Storage Service v2
+  - service: glance
+    type: image
+    description: CIAO Image Service
 
 # CIAO endpoints to create in Keystone
 ciao_openstack_endpoints:
   - service: ciao
     type: compute
     public_url: https://{{ ciao_controller_fqdn }}:8774/v2.1/%(tenant_id)s
-    internal_url: https://{{ ciao_controller_fqdn }}:8774/v2.1/%(tenant_id)s
```

Comment:
Is this something new in openstack playbooks? (to dont put the internal and admin urls)